src/registry.json

```diff
@@ -2078,6 +2087,67 @@
         }
       }
     },
+    "npm:@polkagate/snap": {
+      "id": "npm:@polkagate/snap",
+      "metadata": {
+        "name": "PolkaGate",
+        "author": {
+          "name": "PolkaGate",
+          "website": "https://polkagate.xyz/"
+        },
+        "website": "https://apps.polkagate.xyz/",
+        "summary": "Unlock the Power of Polkadot. Seamlessly manage accounts, connect to Polkadot dApps, and sign transactions.",
+        "description": "The Polkagate Snap is a MetaMask snap designed to enable MetaMask to interact with tokens within the Polkadot ecosystem, including Polkadot, Kusama, and more.\n\nTo use the snap, you must first install MetaMask and create your initial account on Ethereum. This will generate a recovery phrase, which the Polkagate snap will utilize to derive accounts for the Polkadot ecosystem.\n\nAfter MetaMask installation, manage your Polkadot ecosystem accounts and perform various transactions by installing the Polkagate snap. Users can install snaps by visiting the MetaMask Snaps Directory or each snap's specific dApp, which for the Polkagate snap the dapp address is https://apps.polkagate.xyz/ where users can manage transactions such as fund transfers, staking, participating in crowdloans, and more.",
```

Comment:
is a MetaMask **Snap**...

which the Polkagate **Snap** will utilize... 

by installing the Polkagate **Snap**.

> Users can install snaps by visiting the MetaMask Snaps Directory or each snap's specific dApp, which for the Polkagate snap the dapp address is https://apps.polkagate.xyz/ where users can manage transactions such as fund transfers, staking, participating in crowdloans, and more.

This sentence does not make sense in the context of our directory.

Reply:
Done.